contrib/extensions_metadata.yaml

```diff
@@ -53,3 +53,8 @@ envoy.bootstrap.vcl:
   - envoy.bootstrap
   security_posture: requires_trusted_downstream_and_upstream
   status: alpha
+envoy.matching.input_matchers.hyperscan:
+  categories:
+  - envoy.matching.input_matchers
+  security_posture: robust_to_untrusted_downstream
```

Comment:
Does hyperscan have a fuzzer?

Reply:
We have [scripts](https://github.com/intel/hyperscan/tree/master/tools/fuzz) for running fuzz test manually, but we do not have an integrated fuzz test.

Reply:
I'm unsure we should mark as `robust_to_untrusted_downstream` without oss-fuzz integration, so that fuzz tests are running continuously and there is a way to track defects discovered by the fuzzer.

Other than that LGTM

Reply:
Do you think `requires_trusted_downstream_and_upstream` will be better? I will change it in my next commit.